### ISSUE TITLE
Question about the logo

### ISSUE THREAD

**Issue by LitoMore** [xojs/xo#689]
Hi XO team! I'm submitting our XO logo to https://github.com/simple-icons/simple-icons.

As you can see in https://github.com/simple-icons/simple-icons/pull/7749.

The icon in a badge looks not so good. May I increase the stroke-width base on your design for badges? And the ❤️ shape we should make it filled. Like this below:

|Before|After|
|:-:|:-:|
|<img width="725" alt="CleanShot 2022-09-05 at 02 45 37@2x" src="https://user-images.githubusercontent.com/8186898/188328837-a2cc30d6-9710-4dea-99aa-083d40e00a43.png">|<img width="727" alt="CleanShot 2022-09-05 at 03 17 28@2x" src="https://user-images.githubusercontent.com/8186898/188329956-ae951688-a9ec-499c-becd-ecb1510ab711.png">|

**Comment by sindresorhus** [comment]
Yes, you should definitely increase the stroke-width when used in a smaller format like this. 👍

**Comment by LitoMore** [comment]
Now XO is available on https://shields.io.

Here are some examples:

![](https://shields.io/badge/code_style-5ed9c7?logo=xo&logoColor=black)
![](https://shields.io/badge/code_style-5ed9c7?logo=xo&labelColor=gray)
![](https://shields.io/badge/code_style-5ed9c7?logo=xo&labelColor=555)

```markdown
![](https://shields.io/badge/code_style-5ed9c7?logo=xo&logoColor=black)
![](https://shields.io/badge/code_style-5ed9c7?logo=xo&labelColor=gray)
![](https://shields.io/badge/code_style-5ed9c7?logo=xo&labelColor=555)
```

**Comment by sindresorhus** [comment]
Very nice! Would you be interested in submitting a PR to update https://github.com/xojs/xo#badge ?

**Comment by LitoMore** [comment]
@sindresorhus Sure. Which one do you prefer?

**Comment by sindresorhus** [comment]
The second one.
